plugins/Marketplace/Controller.php

```diff
@@ -353,7 +354,7 @@ public function installAllPaidPlugins()
 
             $hasErrors = false;
             foreach ($paidPlugins as $paidPlugin) {
-                if (!$this->canPluginBeInstalled($paidPlugin)) {
+                if (!$this->canPluginBeInstalled($paidPlugin) || $paidPlugin['status'] !== 'Active') {
```

Comment:
Why didn't you put that new check directly in `canPluginBeInstalled`? Above places are the only places where that method is called, and both require the same check...
Also it could be good to introduce constants for the possible status values and use them for comparison

Reply:
@sgiehl Fixed :+1: